### DISCUSSION TITLE
Error saat knitting R markdown ke HTML.

### DISCUSSION THREAD

**Discussion by tu849035** [teamalgoritma/community#228] [Q&A] [answered]
Selamat Sore.
Saat sedang melakukan knitting R markdown ke HTML muncul error seperti tampilan di bawah.

![image](https://user-images.githubusercontent.com/107348211/181219148-00d8c448-5d92-4ef7-8e28-7f7c20cb53d3.png)

Mohon bantuan arahannya.

Terima Kasih

**Comment by idoali** [comment, marked as answer]
Selamat malam pak. Bisa dicoba di restart terlebih dulu R Studio nya pak. Lalu, laporkan lagi semisal masih bermasalah.
